tests/functionality_tests/simple_turn_test.py

```diff
@@ -0,0 +1,32 @@
+# THIS IS A TESTING SCRIPT FOR TURN_TO_TARGET_HEADING. USE THIS SCRIPT WHEN PHYSICALLY TESTING THE ROBOT AND CHANGE THE test_state TO THE STATE YOU NEED
+
+from engine.robot_state import Robot_State
+from engine.robot import Robot
+from engine.robot_logic.traversal import simpler_turn_to_target_heading
+from engine.database import DataBase
+import numpy as np
+import math
+
+if __name__ == "__main__":
+    # 0 does nothing, 1 is a 90 degree turn, 2 is a 180 degree turn, 3 is a -90 degree turn, 4 is a negative 180 degree turn
+    test_state = 0
```

Comment:
you should make it such that all of the test are ran when the file is ran, so test_state should iterate. also, nit, this should be made into an actual unittest file

Reply:
The reason my file is formatted like this is because sensor_physical_testing was the only file in the codebase testing the turn function so I copied it. I will reformat my tests because I think it's a good suggestion but it might take some time.